db/migrations/20190320104850_initial.js

```diff
@@ -1,7 +1,7 @@
 
 exports.up = function(knex, Promise) {
   return Promise.all([
-    knex.schema.createTable('favorites', function(table) {
+    knex.schema.createTable('favorites', (table) => {
```

Comment:
Does this make a difference? I won't request you change this, but could be bike-shedding